### PR TITLE
decouple option value and label

### PR DESCRIPTION
**Pull request by bo-dun-1**

### CODE REVIEW

File: src/utils/formHelperFunctions.ts [hunk 1]
```diff
@@ -76,6 +76,17 @@ function getDefaultFieldValue(field: any) {
   }
 }
 
+function updateStepFieldOptions(step: any, newFieldOptions: any) {
```
Comment (ericpias) [inline src/utils/formHelperFunctions.ts:79]: newFieldOptions: Record<string, string[]> ?

File: src/utils/formHelperFunctions.ts [hunk 2]
```diff
@@ -76,6 +76,17 @@ function getDefaultFieldValue(field: any) {
...
+  step.servar_fields.forEach((field: any) => {
+    const servar = field.servar;
+    if (servar.key in newFieldOptions) {
+      const options = newFieldOptions[servar.key];
+      servar.metadata.options = options;
+      servar.metadata.option_labels = options;
```
Comment (ericpias) [inline src/utils/formHelperFunctions.ts:85]: Should we support setting option labels from the SDK as well?

**Review by ericpias — CHANGES_REQUESTED** [review]
Also, I was expecting that option label would be supported by the dashboard.  I suppose another PR is coming for that?

File: src/utils/formHelperFunctions.ts (hunk 2 above)
Comment (bo-dun-1) [reply src/utils/formHelperFunctions.ts]: added

**Pull request merged**